src/constants/delegates.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export const MAX_BLOCKS_FORGED = 101;
```

Comment:
This is correct for the version 3 of Lisk Core. The v4 and the v5 are going to forge 103 blocks per round.

Reply:
I think yes. We are going to change this constant again when we start using lisk core v4. As far as I know, there is no api that returns this number.

Reply:
That's true, there is no API for that. It can be hardcoded for now, but Lisk Service can provide that value in the future.